calico_containers/calico_ctl/container.py

```diff
@@ -235,8 +234,12 @@ def container_remove(container_name):
                 # enforcing assignments strictly in datastore.py.
                 client.unassign_address(pool, ip)
 
-    # Remove the endpoint
-    netns.remove_veth(endpoint.name)
+    try:
+        # Remove the interface if it exists
+        netns.remove_veth(endpoint.name)
+    except CalledProcessError:
+        print "Could not remove Calico interface"
```

Comment:
I would put the interface name in here, otherwise it doesn't give much to help debug.